numpy/ma/core.pyi

```diff
@@ -465,8 +466,78 @@ class MaskedArray(ndarray[_ShapeType_co, _DType_co]):
     def std(self, axis=..., dtype=..., out=..., ddof=..., keepdims=...): ...
     def round(self, decimals=..., out=...): ...
     def argsort(self, axis=..., kind=..., order=..., endwith=..., fill_value=..., *, stable=...): ...
-    def argmin(self, axis=..., fill_value=..., out=..., *, keepdims=...): ...
-    def argmax(self, axis=..., fill_value=..., out=..., *, keepdims=...): ...
+    @overload
+    def argmin(  # type: ignore[override]
+        self,
+        axis: None = None,
+        fill_value: _ScalarLike_co | None = None,
+        out: None = None,
+        *,
+        keepdims: Literal[False] | _NoValueType = ...,
+    ) -> intp: ...
+    @overload
+    def argmin(  # type: ignore[override]
+        self,
+        axis: _ShapeLike | None = None,
```

Comment:
The docs indicate that `axis` can only be an integer or `None`. And that indeed seems to be the case. So a `SupportsIndex | None` would be more appropriate.